### PR TITLE
docs give clearer instructions for how to download GraphQL schemas

### PR DESCRIPTION
**Pull request by nathggns**

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/eb252554af0f901ff22deebb4dc2c9ad3609578d/index.html

File: README.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ This has been made possible using [Fructose](https://github.com/newsuk/fructose)
 ## Getting Started
 
 1. Run `yarn` to install dependencies with a `GRAPHQL_ENDPOINT` envar set for
-   linting
+   linting. _This is simply an example GraphQL endpoint, and you will need to
+   obtain a valid API endpoint in order to successfully work with this
+   repository._
 
 ```
 GRAPHQL_ENDPOINT=api.io/graphql yarn
```
Comment (colincclark) [inline README.md]: can change this to the prod endpoint instead? So:

```
GRAPHQL_ENDPOINT='https://api.thetimes.co.uk/graphql' yarn

or

GRAPHQL_ENDPOINT='http://localhost:4000/graphql' yarn
```

**Comment by aronshamash** [comment]
I think you need a ` : ` after `docs` in your commit msg

**Comment by nathggns** [comment]
> I think you need a `:` after `docs` in your commit msg

@aronshamash done 👍

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b6490d7bacddfbce71f55b0d43dee2ccf8a3c33b/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/06cd83ec36f60c3bafe72183b41ba4619b2cc8f2/index.html

**Pull request merged**